autocorpus/bioc_formatter.py

```diff
@@ -1,54 +1,26 @@
 """Top-level BioC collection builder script."""
 
-import json
 from datetime import datetime
+from typing import Any
 
 from .bioc_documents import BiocDocument
 
 
-class BiocFormatter:
-    """BioC Collection builder/formatter."""
-
-    def build_bioc_format(self, input_vals):
-        """Constructs a BioC collection from input document-level data.
-
-        Args:
-            input_vals (object): Input document-level data.
-
-        Returns:
-            (dict): BioC collection
-        """
-        return {
-            "source": "Auto-CORPus (full-text)",
-            "date": f"{datetime.today().strftime('%Y%m%d')}",
-            "key": "autocorpus_fulltext.key",
-            "infons": {},
-            "documents": [BiocDocument(input_vals).as_dict()],
-        }
-
-    def __init__(self, input_vals):
-        """Top-level BioC formatter utility.
-
-        Args:
-            input_vals (object): Input document-level data.
-        """
-        self.bioc_output = self.build_bioc_format(input_vals)
-
-    def to_json(self, indent_val=None):
-        """Returns a JSON representation of the BioC collection.
-
-        Args:
-            indent_val (int): level of indentation for the JSON output
-
-        Returns:
-            (str): JSON representation of the BioC collection
-        """
-        return json.dumps(self.bioc_output, indent=indent_val, ensure_ascii=False)
-
-    def to_dict(self):
-        """Returns a dictionary representation of the BioC collection.
-
-        Returns:
-            (dict): dictionary representation of the BioC collection
-        """
-        return self.bioc_output
+def get_formatted_bioc_collection(input_vals: object) -> dict[str, Any]:
+    """Constructs a BioC collection from input document-level data.
+
+    Args:
+        input_vals (object): Input document-level data.
+        json_format (bool): If True, returns the collection as a JSON string.
+
+    Returns:
+        (dict | str): BioC collection
```

Comment:
Return type isn't correct anymore + the `json_format` arg has been removed:

```suggestion
        input_vals: Input document-level data.

    Returns:
        BioC collection
```

Reply:
Corrected the docstring 👍